client/ayon_core/hosts/tvpaint/plugins/publish/collect_workfile_data.py

```diff
@@ -65,7 +65,7 @@ def process(self, context):
         # Collect and store current context to have reference
         current_context = {
             "project_name": context.data["projectName"],
-            "asset_name": context.data["folderPath"],
+            "folder_path": context.data["folderPath"],
```

Comment:
This might not be due to this PR. But **why** is it sometimes `folder_path` key and sometimes `folderPath`? This is bound for confusion. Let's stick to one, no?

Or is there a reason to deviate the two?